docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
 
-1. To start the UI, run the following in `/tutorial_template`:
+The most important file is the Python file that contains the set of definitions for Dagster to load: `jaffle_shop/definitions.py`. Dagster executes the code in this file to find out what assets it should be aware of, as well as details about those assets. For example, when you ran `dagster dev` in the previous step, Dagster executed the code in this file to determine what assets to display in the UI.
 
-   ```shell
-   dagster dev
-   ```
+In order to return a Dagster asset for each dbt model, the code in this file needs to find out what dbt models you have. It finds that out by reading a file called a "manifest.json", which is a file that dbt can generate for any dbt project and contains information about every model, seed, snapshot, test, etc. in the project. Here's the code that does this:
 
-   Which will result in output similar to:
 
-   ```shell
-   Serving dagster-webserver on http://127.0.0.1:3000 in process 70635
-   ```
+```python
+dbt_project_dir = Path(__file__).parent.joinpath("..", "..")
+dbt = DbtCliResource(project_dir=os.fspath(dbt_project_dir))
+
+# If DAGSTER_DBT_PARSE_PROJECT_ON_LOAD is set, a manifest will be created at runtime.
+# Otherwise, we expect a manifest to be present in the project's target directory.
+if os.getenv("DAGSTER_DBT_PARSE_PROJECT_ON_LOAD"):
+    dbt_parse_invocation = dbt.cli(["parse"], manifest={}).wait()
+    dbt_manifest_path = dbt_parse_invocation.target_path.joinpath("manifest.json")
+else:
+    dbt_manifest_path = dbt_project_dir.joinpath("target", "manifest.json")
+```
 
-2. In your browser, navigate to <http://127.0.0.1:3000>. The page will display the assets:
+As the comment explains, the code gives you a choice about how to create this manifest.json file. Based on an environment variable, either:
+1. This code generates the manifest.json for you.
+2. This code leaves it up to you to generate the manifest.json file on your own, and this code just reads it.
 
-   <Image
-   alt="Asset graph in the UI, containing dbt models loaded as Dagster assets"
-   src="/images/integrations/dbt/using-dbt-with-dagster/load-dbt-models-dbt-asset-graph.png"
-   width={1090}
-   height={894}
-   />
+The first option is the easiest during development, because you never need to worry about the file being out-of-date with your dbt project. But generating the file can be time-consuming when you have a large dbt project, so it's generally recommended to go with the second option when you're deploying your dbt project to production. The second option allows you to only generate it once every time you deploy, rather than every time the code in this module gets executed.
+
+In this tutorial, we're doing with the first option: When we ran `dagster dev`, we set the DAGSTER_DBT_PARSE_PROJECT_ON_LOAD environment variable.
+
+Once we've got a path to a manifest.json, it's time to define our Dagster assets using it. Here's the code that does that:
```

Comment:
```suggestion
Once you've got a path to a `manifest.json`, you can define your Dagster assets using it. For example:
```

Suggesting using `you` instead of `we` here. This reads as more instructive than explanatory to me, so IMO using `you` is a better fit. This would also make the usage consistent with the rest of the tutorial.

Also suggesting an alternative for `Here's the code that does that:` - it feels nitpicky on my part, but it reads as a bit too casual for me. Thoughts?